common/dex-auth/overlays/README.md

```diff
@@ -0,0 +1,44 @@
+# Editing Overlay files
```

Comment:
This file needs cleaning up along with overlays folder.

After parameterization, the only overlay applicable should be for specifying [connector configuration](https://github.com/kubeflow/manifests/pull/195/files#diff-237d2c060bba4c6361e74fc87f56bfc7R20) for Dex auth server.